### PR TITLE
Update readmes v3

### PR DESCRIPTION
**Pull request by hectormmg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
 
-- [Microsoft Authentication Library for Node.js v1.x](lib/msal-node/): A [Node.js](https://nodejs.org/en/) library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the following OAuth 2.0 protocols and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc):
-  - [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with [PKCE](https://oauth.net/2/pkce/)
-  - [Device Code Grant](https://oauth.net/2/grant-types/device-code/)
-  - [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
-  - [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
+-   [Microsoft Authentication Library for Node.js v2.x](lib/msal-node/): A [Node.js](https://nodejs.org/en/) library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the following OAuth 2.0 protocols and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc):
```
Comment (tnorling) [inline README.md]: Let's take the versioning off this so we don't need to maintain it
```suggestion
-   [Microsoft Authentication Library for Node.js](lib/msal-node/): A [Node.js](https://nodejs.org/en/) library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the following OAuth 2.0 protocols and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc):
```

File: README.md [hunk 2]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+    -   [Device Code Grant](https://oauth.net/2/grant-types/device-code/)
+    -   [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+    -   [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
+    -   [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
+    -   [On-behalf-of Flow](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow)
 
-- [Microsoft Authentication Library for React v1.x](lib/msal-react/): A wrapper of the msal-browser 2.x library for apps using React.
+-   [Microsoft Authentication Library for JavaScript v3.x](lib/msal-browser/): A browser-based, framework-agnostic browser library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the OAuth 2.0 [Authorization Code Flow with PKCE](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow), and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc).
```
Comment (tnorling) [inline README.md]:
```suggestion
-   [Microsoft Authentication Library for JavaScript](lib/msal-browser/): A browser-based, framework-agnostic browser library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the OAuth 2.0 [Authorization Code Flow with PKCE](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow), and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc).
```

File: README.md [hunk 3]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+-   [Microsoft Authentication Library for React v2.x](lib/msal-react/): A wrapper of the msal-browser 2.x library for apps using React.
```
Comment (tnorling) [inline README.md]:
```suggestion
-   [Microsoft Authentication Library for React](lib/msal-react/): A wrapper of the msal-browser 2.x library for apps using React.
```

File: README.md [hunk 4]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+-   [Microsoft Authentication Library for Angular v3.x](lib/msal-angular/): A wrapper of the msal-browser 2.x library for apps using Angular framework.
```
Comment (tnorling) [inline README.md]:
```suggestion
-   [Microsoft Authentication Library for Angular](lib/msal-angular/): A wrapper of the msal-browser 2.x library for apps using Angular framework.
```

File: README.md [hunk 5]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+-   [Microsoft Authentication Extensions for Node v1.x](extensions/msal-node-extensions/): The Microsoft Authentication Extensions for Node offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. It gives additional support to the Microsoft Authentication Library for Node (MSAL).
```
Comment (tnorling) [inline README.md]:
```suggestion
-   [Microsoft Authentication Extensions for Node](extensions/msal-node-extensions/): The Microsoft Authentication Extensions for Node offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. It gives additional support to the Microsoft Authentication Library for Node (MSAL).
```

File: lib/msal-node/README.md [hunk 6]
```diff
@@ -29,32 +29,36 @@ MSAL Node enables applications to authenticate users using [Azure AD](https://do
...
+-   [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+-   [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
+-   [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
+-   [On-behalf-of flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow)
+-   [Username and Password flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#usernamepassword)
 
 **[Coming Soon]** In the future we plan to add support for:
-- [Integrated Windows Authentication flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#integrated-windows-authentication)
+
+-   [Integrated Windows Authentication flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#integrated-windows-authentication)
```
Comment (tnorling) [inline lib/msal-node/README.md]: These can probably be removed

**Review by tnorling — APPROVED** [review]

File: lib/msal-node/README.md (hunk 6 above)
Comment (hectormmg) [reply lib/msal-node/README.md]: Just delete the whole thing or just the Coming soon?

Comment (tnorling) [reply lib/msal-node/README.md]: Lines 48-50

File: README.md [hunk 7]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+    -   [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+    -   [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
+    -   [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
+    -   [On-behalf-of Flow](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow)
 
-- [Microsoft Authentication Library for React v1.x](lib/msal-react/): A wrapper of the msal-browser 2.x library for apps using React.
+-   [Microsoft Authentication Library for JavaScript](lib/msal-browser/): A browser-based, framework-agnostic browser library that enables authentication and token acquisition with the Microsoft Identity platform in JavaScript applications. Implements the OAuth 2.0 [Authorization Code Flow with PKCE](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow), and is [OpenID-compliant](https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc).
+-   [Microsoft Authentication Library for React](lib/msal-react/): A wrapper of the msal-browser 2.x library for apps using React.
```
Comment (jo-arroyo) [inline README.md]: Should this say browser 3.x library?

File: README.md [hunk 8]
```diff
@@ -4,37 +4,32 @@ The Microsoft Authentication Library for JavaScript enables both client-side and
...
+-   [Microsoft Authentication Library for Angular](lib/msal-angular/): A wrapper of the msal-browser 2.x library for apps using Angular framework.
```
Comment (jo-arroyo) [inline README.md]: Same as React above.

File: README.md (hunk 7 above)
Comment (hectormmg) [reply README.md]: As @tnorling pointed out if we don't add the version here we don't have to update this with every major verison.

Comment (hectormmg) [reply README.md]: Oh never mind it's the description text, I'll remove the version there as well.

**Review by peterzenz — APPROVED** [review]

**Pull request merged**